matlab/schemas/+vis/FancyBar.m

```diff
@@ -1,18 +1,20 @@
  %{
-vis.FancyBar (manual) # pink noise with periods of motion and orientation$
+vis.FancyBar (manual) # Moving Bar stimulus that keeps the size and speed
```

Comment:
okay, then let's add "in spherical coordinates centered on the subject's eye" to make it clear.